packages/flame/lib/src/game/game_widget/game_widget.dart

```diff
@@ -138,11 +138,15 @@ class _GameWidgetState<T extends Game> extends State<GameWidget<T>> {
 
   MouseCursor? _mouseCursor;
 
-  Future<void> get _loaderFuture {
-    final onLoad = widget.game.onLoadCache;
-    final onMount = widget.game.onMount;
-    return (onLoad ?? Future<void>.value()).then((_) => onMount());
-  }
+  Future<void> get onloadFuture =>
```

Comment:
```suggestion
  Future<void> get loaderFuture =>
```

Reply:
If it was by me I would name it as claudinho